lib/reporter/reporters/html.js

```diff
@@ -111,7 +111,7 @@ class HtmlReporter extends BaseReporter {
       this.environments[testEnv].stats.failed += failedCount;
       this.environments[testEnv].stats.skipped += module.skippedCount;
       this.environments[testEnv].stats.total += passedCount + failedCount + module.skippedCount;
-      this.environments[testEnv].stats.time += this.environments[testEnv].stats.time || module.timeMs;
+      this.environments[testEnv].stats.time += module.timeMs || this.environments[testEnv].stats.time;
```

Comment:
why do we need this again?

Reply:
This change was made as in the `if` condition, we initialize the object and declare `this.environments[testEnv].stats.time` value. When iterating it for other modules, we need to add those module times, not the time we declared initially.

Nice catch!  We can safely remove this line since we removed the build time from the Environment Dropdown.

Reply:
cool